src/components/classItem/classItem.style.scss

```
@@ -0,0 +1,111 @@
+@use "../../style/variables" as var;
+@use "../../style/mixins.scss" as mixin;
+
+.class-item {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-radius: 4px;
+  padding: 8px 4px;
+  width: 72px;
+  height: 64px;
+  text-align: center;
+  cursor: pointer;
+
+  &--empty {
+    cursor: default;
+  }
+
+  &--default {
+    background-color: var.$blue-darker;
+    border: 1px solid var.$divider-color;
+    &:hover {
+      box-shadow: 0px 0px 2px 0px var.$blue-weak-emphasis;
+    }
+    &:active {
+      box-shadow: none;
+      opacity: 80%;
+    }
+  }
+
+  &--disabled {
+    background-color: var.$black;
+    border: 1px dashed var.$divider-color;
+    &:hover {
+      border-color: var.$blue-weak-emphasis;
+    }
+    &:active {
+      border-color: var.$divider-color;
+      opacity: 80%;
+    }
+  }
+
+  &--selected {
+    background-color: var.$blue-darker;
+    border: 1px solid var.$blue-gray;
+    &:hover {
+      box-shadow: 0px 0px 2px 0px var.$blue-weak-emphasis;
+    }
+    &:active {
+      box-shadow: none;
+      opacity: 80%;
+    }
+  }
+
+  &__checkbox {
+    position: absolute;
+    top: 5.1px;
+    right: 5.1px;
+    width: 8px;
+    height: 8px;
+    border-radius: 1px;
+    color: var.$blue-darker;
+    font-size: 1.5rem;
+    margin-bottom: 0.5rem;
+
+    &--default {
+      background-color: var.$blue-darker;
+      border: 1px solid var.$divider-color;
+    }
+    &--selected {
+      background-color: var.$blue-light-gray;
+      border: 1px solid var.$blue-light-gray;
+    }
+  }
+
+  &__title {
+    color: var.$blue-white;
+    font-family: Montserrat;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0%;
+    text-align: center;
+  }
+  &__description {
+    color: var.$blue-gray;
+    font-family: Montserrat;
+    font-weight: 500;
+    font-size: 6px;
+    line-height: 122%;
+    letter-spacing: 0%;
+    text-align: center;
+
+    @include mixin.dynamicLineClamp;
+  }
+  &__credit {
+    color: var.$blue-gray;
+    font-family: Montserrat;
+    font-weight: 500;
+    font-size: 6px;
+    line-height: 122%;
+    letter-spacing: 0%;
+    text-align: center;
+
+    position: absolute;
+    bottom: 5px;
+    right: 5.5px;
+  }
+}
```

Todas os textos do código ja possuem fonte, letter-spacing e text-align. Remover para evitar repetição de código.

Além disson, a região de credit tem que estar 4px do right e do bottom, ao invés do 5.5px e 5px atuais

```suggestion
  &__title {
    color: var.$blue-white;
    font-weight: 500;
    font-size: 12px;
    line-height: 16px;
  }
  &__description {
    color: var.$blue-gray;
    font-weight: 500;
    font-size: 6px;
    line-height: 122%;

    @include mixin.dynamicLineClamp;
  }
  &__credit {
    color: var.$blue-gray;
    font-weight: 500;
    font-size: 6px;
    line-height: 122%;
    position: absolute;
    bottom: 4px;
    right: 4px;
  }
}
```